source/includes/fact-favorite-query.rst

```diff
@@ -1,29 +1,15 @@
 .. procedure::
    :style: normal
 
-   .. step:: Open query history
+   .. step:: Open query history.
 
       Click the :guilabel:`History` button at the top of the pipeline 
       builder.
```

Comment:
Not from these changes, but I'm thinking we can fix this up here: 
```suggestion
      Click the :guilabel:`History` button on the query bar.
```
The `query/recent/#view-recent-queries` page or the screenshot below in `query/queries/#using-the-favorites-tab` might be nice to reference here as well.

Reply:
Done.